From f3cf49df860d429af49de992cdf577a16dcab5b1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:44:35 +0100
Subject: [PATCH 039/133] std.Target: bump contiki max version to 5.1.0
X-Developer-Signature: v=1; a=openpgp-sha256; l=727; i=xtex@astrafall.org;
 h=from:subject; bh=cc0u6zcknKPIDC/K4S4sy4ZOiXa+lT6c2fo/wPEAKYE=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv/9ceso7b/l+/C2xPHWkguTvJ2KZwcX+2ot/Hyx9
 U9osYxnx0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBUV5xlZGiIWFfsanbAvfDq3+Zdr9uezc1/eTRm8pupsuJ/bfrTDgYx/FM7z5T
 M3xH92uRz68rpE5qnyHvFhdUwfH+eeyz4WebCbFYA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 38413446b06b6b7d63bcd67e5dcf3ac5fc053b0d)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 524fe1cf8926..99e2a3af6125 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -400,7 +400,7 @@ pub const Os = struct {
                 .contiki => .{
                     .semver = .{
                         .min = .{ .major = 4, .minor = 0, .patch = 0 },
-                        .max = .{ .major = 5, .minor = 0, .patch = 0 },
+                        .max = .{ .major = 5, .minor = 1, .patch = 0 },
                     },
                 },
                 .fuchsia => .{
-- 
2.52.0

